docs/installation/manual/installing_karaf_and_alpaca.md

```diff
@@ -335,12 +365,6 @@ Before we can configure the features we’re going to use, they need to be insta
 # we're using, but is being included here to resolve the aforementioned
 # missing link in the dependency chain.
 /opt/karaf/bin/client feature:install jena
-/opt/karaf/bin/client feature:install fcrepo-camel
```

Comment:
Why was this, and the rest of these removed? I needed to run them for my site to work.

Reply:
I was constantly getting a version mismatch error with that on Debian 10. I'll try again with a clean install. If it works for you you can revert that chunk.

Reply:
Perhaps the version mismatch errors you experienced could be resolved by using the versions I mentioned in the above comments (i.e. keeping Camel @ 2.x and Jena @ 3.x)?

Reply:
I'll be sure to try that out soon and get back to you. That could indeed be it.